docs/usage/MigPlanPVSelection.md

```diff
@@ -0,0 +1,97 @@
+| accessMode   | The access mode to use for the PVC in the destination cluster. If empty, the source cluster PVC's access mode will be used.   | <empty> | "ReadWriteOnce", "ReadWriteMany" |
+| verify       | Whether or not to perform filesystem copy verification after copying volume contents. This is only applicable to filesystem copy. | false | true, false |
+
+Note that `accessMode` is not currently exposed via the web user
+interface, which means that editing the MigPlan yaml is the only way
+to set this value.
+
+The `supported` field will list the available actions and copy methods
+for a given volume. The `selection` value for these fields must be
+found there. For `storageClass`, the selection must be found in the
+list from `migplan.status.destStorageClasses`.
+
+## Other persistent volume fields
+
+The full list of migplan `persistentVolumes` fields is as follows:
+
+| Field             | Description |
+|-------------------|-------------|
+| name              | The name of the source PV |
+| capacity          | The capacity of the source PV |
+| storageClass      | The source PV storageClass |
+| supported         | Supported actions and copy methods for the volume |
+| selection         | User selections for the volume |
+| pvc               | Source PVC metadata (see below for fields) |
+| nfs               | NFS volumesource information copied from PV for NFS volumes |
+| staged            | Internal field used for managing updates |
+| proposedCapacity  | Used for PV resizing for nearly-full volumes |
+| capacityConfirmed | Used for PV resizing for nearly-full volumes |
+
+`pvc` subfields:
+
+| Field        | Description |
+|--------------|-------------|
+| namespace    | The PVC namespace |
+| name         | The PVC name |
+| accessModes  | PVC accessModes |
+| hasReference | Whether or not this PVC is referenced by pods or pod template resources |
+
+## Storageclass status fields
+
+`status.srcStorageClasses` and `status.destStorageClasses` follow the
+same format. The value is a slice of storageclass values. The
+`destStorageClasses` value is the one which is relevant to volume
+selection. `selection.storageClass` must match the `name` field in one
+of the values in `status.destStorageClasses`. The storage class subfields are as follows:
+
+| Field       | Description |
+|-------------|-------------|
+| Name        | The name of the storage class |
+| Provisioner | The provisioner used by the storage class |
+| Default     | Whether or not this is the default storage class |
+| AccessModes | A list of access modes that are known to be supported by the storage class (may not be complete) |
```

Comment:
@apinnick Above here are the fields for the storage class struct that are filled in for storageclasses (both srcStorageClasses and destStorageClasses, although it's the dest one that matters for users filling out pv selection in terms of finding valid values)